dadb/src/test/kotlin/dadb/TestUtils.kt

```diff
@@ -39,6 +39,12 @@ fun killServer() {
     } catch (ignore: IOException) {}
 }
 
+fun startServer() {
+    try {
+        Runtime.getRuntime().exec("adb start-server").waitFor()
```

Comment:
Should `AdbServerDadb` start server on its own if it is not running?

Reply:
Yeah that's a good ideax